winml/dll/module.cpp

```diff
@@ -85,6 +86,35 @@ STDAPI DllCanUnloadNow() {
   return S_FALSE;
 }
 
+int32_t WINRT_CALL WinmlMoreGetActivationFactory(void* classId, void** factory) noexcept {
+  try {
+    *factory = nullptr;
+    uint32_t length{};
+    wchar_t const* const buffer = WINRT_WindowsGetStringRawBuffer(classId, &length);
+    std::wstring_view const name{buffer, length};
+
+    auto requal = [](std::wstring_view const& left, std::wstring_view const& right) noexcept {
+      return std::equal(left.rbegin(), left.rend(), right.rbegin(), right.rend());
+    };
+
+    if (requal(name, L"Microsoft.AI.MachineLearning.Experimental.Dummy")) {
+      *factory = winrt::detach_abi(winrt::make<winrt::Microsoft::AI::MachineLearning::Experimental::factory_implementation::Dummy>());
+      return 0;
+    }
+
+#ifdef _WRL_MODULE_H_
```

Comment:
What is the purpose of Line 105 - Line 112 ?

Reply:
I have remove them, it is boilerplate to allow for wrl implemented winrt components (which we don't have)